intercom_flutter/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Jun 23 08:50:38 CEST 2017
+#Thu Jun 17 14:27:11 IST 2021
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
Latest version is `7.1`, again it's for the example but maybe we should jump to latest available

Reply:
no need for now.